tests/test_conditioning.py

```
@@ -0,0 +1,164 @@
+    def test_invalid_sample_rate_raises(self):
+        with pytest.raises(ValueError):
+            mpdsp.PeakEnvelope(sample_rate=-1.0,
+                                attack_ms=5.0, release_ms=50.0)
+
+    def test_non_positive_attack_raises(self):
+        with pytest.raises(ValueError):
+            mpdsp.PeakEnvelope(sample_rate=SAMPLE_RATE,
+                                attack_ms=0.0, release_ms=50.0)
+
+    def test_unknown_dtype_raises(self):
+        with pytest.raises(Exception):
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)